### PR TITLE
Add projects from front-end

### PR DESCRIPTION
**Pull request by ishaaa-ai**
- Created ProjectsController.js file
- Changed routes.js (since ProjectsController.js was created)
- Made changes to Post-Projects.vue so that projects can be added from the front-end.
- Changed Project.js to correct syntax errors

### CODE REVIEW

File: client/src/components/PostProjects.vue [hunk 1]
```diff
@@ -58,33 +58,30 @@
...
-        },
-        computed: {
-            availableOptions() {
-                return this.options.filter(opt => this.value.indexOf(opt) === -1)
+                name: '',
+                description: ''
             }
         },
+        // computed: {
+        //     availableOptions() {
```
Comment (VeerpalBrar) [inline client/src/components/PostProjects.vue]: Why is this commented out? It looks like it's used on line 40. I think it's okay to have tag stuff on the front end even if it's not connected to the back end yet. We can do that in another PR.  I only mention this here because I'm wondering if the form might error out if it can't find `avaliableOptions()`

File: client/src/components/PostProjects.vue [hunk 2]
```diff
@@ -58,33 +58,30 @@
 <script>
     import axios from 'axios'
     export default {
-        name:"submit",
+        name:"addProject",
         data() {
             return {
-                options: ['Computer Science', 'Java', 'A.I.', 'Machine Learning', 'Python'],
```
Comment (VeerpalBrar) [inline client/src/components/PostProjects.vue]: if you choose to uncomment the avaliableOptions code, you'll have to add these two lines back to the file.

Comment (ishaaa-ai) [reply client/src/components/PostProjects.vue]: Okay i'll add them back!
